### PR TITLE
Add invalid repo test.

### PR DESCRIPTION
**Pull request by etefera**
Ensure error is displayed when repo doesn't exist.
https://github.com/GoogleContainerTools/kpt/issues/1682

### CODE REVIEW

File: internal/util/update/update_test.go [hunk 1]
```diff
@@ -678,6 +678,46 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
 	}
 }
 
+// TestCommand_Run_failInvalidRepo verifies Run fails if the repo is invalid
+func TestCommand_Run_failInvalidRepo(t *testing.T) {
+	for _, strategy := range kptfilev1alpha2.UpdateStrategies {
```
Comment (mortent) [inline internal/util/update/update_test.go:683]: We don't really need to test for each strategy here. This functionality is independent of which strategy is being used.

File: internal/util/update/update_test.go [hunk 2]
```diff
@@ -678,6 +678,46 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
...
+		t.Run(string(strategy), func(t *testing.T) {
+			g := &testutil.TestSetupManager{
+				T: t,
+				ReposChanges: map[string][]testutil.Content{
+					testutil.Upstream: {
+						{
+							Data:   testutil.Dataset1,
```
Comment (mortent) [inline internal/util/update/update_test.go:690]: We are trying to move more towards the `pkgbuilder` pattern, as using golden data sets becomes difficult when there are multiple repos where each has multiple commits. But for this test the content doesn't really matter, so it is ok here.

File: internal/util/update/update_test.go [hunk 3]
```diff
@@ -678,6 +678,46 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
...
+				return
+			}
+
+			err := Command{
+				Pkg:      pkgtest.CreatePkgOrFail(t, g.LocalWorkspace.FullPackagePath()),
+				Repo:     "fake",
+				Strategy: strategy,
+			}.Run()
+			if !assert.Error(t, err) {
+				return
```
Comment (mortent) [inline internal/util/update/update_test.go:710]: nit: We usually use `t.FailNow()` in these situations.

File: internal/util/update/update_test.go (hunk 1 above)
Comment (etefera) [reply internal/util/update/update_test.go:683]: Removed.

File: internal/util/update/update_test.go (hunk 2 above)
Comment (etefera) [reply internal/util/update/update_test.go:690]: I'll start basing other tests on that pattern.

File: internal/util/update/update_test.go (hunk 3 above)
Comment (etefera) [reply internal/util/update/update_test.go:710]: Replaced.

**Review by mortent — APPROVED** [review]

**Pull request merged**